src/Core/Models/Api/Request/CipherRequestModel.cs

```diff
@@ -59,6 +60,7 @@ public CipherDetails ToCipherDetails(CipherDetails existingCipher)
         {
             existingCipher.FolderId = string.IsNullOrWhiteSpace(FolderId) ? null : (Guid?)new Guid(FolderId);
             existingCipher.Favorite = Favorite;
+            existingCipher.Reprompt = Reprompt == CipherRepromptType.None ? null: (CipherRepromptType?)Reprompt;
```

Comment:
I think it's okay to save None there if a value is set/passed in this case, the idea is just that `null` is allowed and indicates "not set". I think you can take out this ternary assignment and just use `Reprompt` in this set. Otherwise, you would at the least need to fix the formatting to add a space before the ` : ` in the ternary statement.